### PR TITLE
chore: update routes

### PR DESCRIPTION
**Pull request by catngh**
**What does this PR do?**

- update routes

### CODE REVIEW

File: src/adapters/config.ts [hunk 1]
```diff
@@ -791,7 +808,7 @@ class Config extends Fetcher {
     symbol: string
     chain_id: number
   }) {
-    return await this.jsonFetch(`${API_BASE_URL}/configs/default-nft-ticker`, {
+    return await this.jsonFetch(`${API_BASE_URL}/nfts/default-nft-ticker`, {
```
Comment (tuanddd) [inline src/adapters/config.ts:811]: `nfts` or `nft` ?

File: src/adapters/profile.ts [hunk 2]
```diff
@@ -41,7 +41,7 @@ class Profile extends Fetcher {
 
   public async getUserProfile(guildId: string, userId: string) {
     return await this.jsonFetch<ResponseGetDataUserProfileResponse>(
-      `${API_BASE_URL}/profiles`,
+      `${API_BASE_URL}/users/profiles`,
```
Comment (tuanddd) [inline src/adapters/profile.ts:44]: `users` or `user`?

File: src/adapters/config.ts (hunk 1 above)
Comment (catngh) [reply src/adapters/config.ts:811]: the same group as other `/nfts` routes

File: src/adapters/profile.ts (hunk 2 above)
Comment (catngh) [reply src/adapters/profile.ts:44]: the same group as other `/users` routes

**Review by tuanddd — APPROVED** [review]

**Review by hoangharry — APPROVED** [review]

**Pull request merged**